javascript/jsonwebtoken/security/jwt-exposed-credentials.yaml

```diff
@@ -4,57 +4,52 @@ rules:
       - javascript
       - typescript
     metadata:
-      cwe: "CWE-522: Insufficiently Protected Credentials"
-      owasp: "A2: Broken Authentication"
-      source-rule-url: https://r2c.dev/blog/2020/hardcoded-secrets-unverified-tokens-and-other-common-jwt-mistakes/
+      cwe: "CWE-798: Use of Hard-coded Credentials"
+      references: 
+        - https://cheatsheetseries.owasp.org/cheatsheets/Secrets_Management_CheatSheet.html
+      owasp:
+        - A07:2021 - Identification and Authentication Failures
+        - A03:2017 - Sensitive Data Exposure
       asvs:
         section: "V3: Session Management Verification Requirements"
         control_id: 3.5.2 Static API keys or secret
         control_url: https://github.com/OWASP/ASVS/blob/master/4.0/en/0x12-V3-Session-management.md#v35-token-based-session-management
         version: "4"
-      references:
-        - https://cwe.mitre.org/data/definitions/522.html
       category: security
       technology:
         - jwt
-    message: >-
-      Password is exposed through JWT token payload. This is not encrypted and
-      the password could be compromised. Do not store passwords in JWT tokens.
+        - secrets
+    message: >- 
+      A hard-coded credential was detected. It is not recommended 
```

Comment:
See other comment re: exposed-credential rules. I think this was intended to be an info disclosure rule, not a hardcoded secret rule.